### PR TITLE
Feat(CX-3361): redesign the swa landing page part 1

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3361]

### Description

https://user-images.githubusercontent.com/36167539/217002638-69d3669b-8fde-40fd-859d-ba5f11d796dc.mov

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3361]: https://artsyproduct.atlassian.net/browse/CX-3361?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dariakoko** [comment]
@MounirDhahri the design was updated, here is the latest Figma link https://www.figma.com/file/WgcIDeIJkytjcCRkpEcUE9/Sell-with-Artsy---CX?t=y6WtD6xpB6EKbrf5-0

**Comment by dariakoko** [comment]
Some things will have to be changed too (images, texts), but in the end of the week, we will catch up with Dasha later, when the changes are approved

File: src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx [hunk 1]
```diff
@@ -34,7 +42,21 @@ export const MarketingLandingApp = () => {
         sessionStorage.setItem("utmParams", JSON.stringify(queryUtmParams))
       }
     }
-  }, [])
+  }, [query.utm_medium, query.utm_source, query.utm_term])
```
Comment (dariakoko) [inline src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx:45]: fyi: couldn't commit without this change

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { color, Icon, IconProps, Path, Title } from "@artsy/palette"
+
+export const DollarCircleIcon: React.FC<IconProps> = ({
+  title = "Dollar in circle",
+  ...props
+}) => {
+  return (
+    <Icon {...props} viewBox="0 0 50 50">
+      <Title>{title}</Title>
+      <Path
```
Comment (MounirDhahri) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon.tsx:10]: I would recommend adding the icon to palette along with the exisitng `MoneyFillIcon` https://palette.artsy.net/atoms/icons/#gatsby-focus-wrapper

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import { useAnalyticsContext, useSystemContext } from "System"
+import { useTracking } from "react-tracking"
+
+export const HeaderSWA = () => {
+  const enableSWAInquiryFlow = useFeatureFlag("swa-inquiry-flow")
+  const { user } = useSystemContext()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const { trackEvent } = useTracking()
+
+  const getInTouchRoute = enableSWAInquiryFlow
```
Comment (MounirDhahri) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx:22]: **praise:** 👏

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      context_page_owner_type: contextPageOwnerType,
+      label: "Get in Touch",
+      user_id: user?.id,
+      user_email: user?.email,
+    })
+  }
+
+  const image = resized(
+    "https://files.artsy.net/images/content-card-swa-landing-page.png",
+    { width: 1104, height: 833 } // TODO: Source image should be 4:3, ideally
```
Comment (MounirDhahri) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx:50]: **issue(non-blocker):** is there something we can do about this already? if not we can remove the todo

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/WaysWeSell.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+  srcSet: string
+  text?: string
+  title?: string
+  useBlackBackground?: boolean
+  imageHeight?: number
+}
+
+import { resized } from "Utils/resized"
+
+const imageWidth = 600
```
Comment (MounirDhahri) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/WaysWeSell.tsx:15]: **nitpick:** 
```suggestion
const IMAGE_WIDTH = 600
```

**Review by MounirDhahri — APPROVED** [review]
LGTM, I left one comment about moving the icons to palette, the rest are not blocking

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx (hunk 4 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx:50]: That's not the final image, I'd keep it as a reminder and remove when the design is finalised

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon.tsx:10]: Thanks Daria - sounds great

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx (hunk 4 above)
Comment (MounirDhahri) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx:50]: cool - thanks

**Comment by MounirDhahri** [comment]
LGTM @dariakoko please feel free to merge

**Review by MounirDhahri — APPROVED** [review]

File: src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx (hunk 1 above)
Comment (lordkiz) [reply src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx:45]: FYI: you can always disable exhaustive deps rule for that line.

**Comment by MounirDhahri** [comment]
🚀

**Pull request merged**